WMF Framework/WMFContentGroup+Extensions.m

```diff
@@ -82,6 +85,7 @@ - (void)updateContentType {
             self.contentType = WMFContentTypeTopReadPreview;
             break;
         case WMFContentGroupKindNews:
+        case WMFContentGroupKindOnThisDay:
```

Comment:
Would be good to make this a separate content type since it's a different content model object